typings.d.ts

```diff
@@ -270,8 +270,8 @@ declare module 'resin-components' {
 			title?: string;
 			titleElement?: string | JSX.Element;
 			titleDetails?: string | JSX.Element;
-			action: string;
-			cancel: () => any;
+			action: string | JSX.Element;
```

Comment:
Why do we want to allow JSX elements here?

Reply:
So that we can have buttons with icons & text.

Reply:
Ack sounds good